applications/GeoMechanicsApplication/custom_utilities/constitutive_law_utilities.hpp

```diff
@@ -24,6 +25,14 @@ class ConstitutiveLawUtilities
 public:
     static int GetStateVariableIndex(const Variable<double>& rThisVariable);
 
+    static void SetSixMechanicsConstitutiveParameters(ConstitutiveLaw::Parameters& rConstitutiveParameters,
```

Comment:
Perhaps a name like `SetConstitutiveParameters` will also do?

Reply:
Restored the old title.